modules/network/pre-existing-subnetwork/README.md

```diff
@@ -0,0 +1,93 @@
+## Description
+
+This module is inspired by pre-existing-vpc, but is changed with the objective of supporting Shared VPCs.
```

Comment:
Let's omit "inspired by" and "shared VPC" parts.

The whole line can be replaced with

> This module discovers a subnetwork that already exists in Google Cloud and
outputs subnetwork attributes that uniquely identify it for use by other modules.